GETTING_STARTED.md

```diff
@@ -0,0 +1,42 @@
+# Getting started
+Before you start, you should understand the concept of forward/up and reverse/down database migrations.
+
+Configure a database for your application. Make sure that your database driver is supported [here](README.md#databases)
+
+## Create migrations
+Create some migrations using migrate CLI. Here is an example:
+```
+migrate create -ext sql -dir db/migrations -seq create_users_table
+```
+Once you create your files, you should fill them.
+
+**IMPORTANT:** In a project developed by more than one person there is a small probability of migrations incosistency - e.g. two developers can create conflicting migrations, and the developer that created his migration later gets it merged to the repository first.
+Keep an eye on such cases (and be even more careful when cherry picking).
```

Comment:
Remove reference to cherry picking since that doesn't increase the likelihood of an inconsistency occurring. I'd just note that developers and teams should keep any eye out for this issue, especially during code review. A note or link to [this summary](https://github.com/golang-migrate/migrate/issues/179#issuecomment-475821264) would be nice too. Namely it'd be good to get more input on the what's the best solution for this problem.

Reply:
Ok. What I meant here is that someone could cherry pick changes including migration dependent on some earlier migrations.